docs/stransmit.md

```diff
@@ -164,15 +164,21 @@ example, **-t:60**. Alternatively you can also separate them by a space,
 but this space must be part of the parameter and not extracted by a
 shell (using quotes or backslash).
 
-- **-timeout, -t, -to** - Sets the timeout for any activity from any medium (in seconds). Default is 30. Use -1 for infinite (that is, turn this mechanism off). The mechanism is such that the SIGALRM is set up to be called after the given time and it's reset after every reading succeeded. When the alarm expires due to no reading activity in defined time, it will break the application. **Notes:**
+- **-timeout, -t, -to** - Sets the timeout for any activity from any medium (in seconds). Default is 0 for infinite (that is, turn this mechanism off). The mechanism is such that the SIGALRM is set up to be called after the given time and it's reset after every reading succeeded. When the alarm expires due to no reading activity in defined time, it will break the application. **Notes:**
     - The alarm is set up after the reading loop has started, **not when the application has started**. That is, a caller will still wait the standard timeout to connect, and a listener may wait infinitely until some peer connects; only after the connection is established is the alarm counting started. 
     - **The timeout mechanism doesn't work on Windows at all.** It behaves as if the timeout was set to **-1** and it's not modifiable.
+- **-timeout-mode, -tm** - timeout mode used. Default is 0 - timeout will happen after the specified time. Mode 1 cancels the timeout if the connection was established.
```

Comment:
Unclear and misleading. Why not introduce an option for "connection timeout" instead? Moreover, it should be better explained how it applies to listener and the caller because for caller you have also `conntimeo` option.

Reply:
Might be a good alternative. Although this `-tm` option is already in master after PR #570. 
But the idea with `timeout-mode` was also to add timeout mode 2, that will start after the connection is established. This will be useful for automated testing.
I suggest to make a separate issue for that, if we want to have another name. This PR is to clean up the command-line parsing and add CSV stats.

Reply:
Ok, so let's leave it for later then.